tests/lookup/tests.py

```diff
@@ -472,7 +472,7 @@ def test_error_messages(self):
             self.fail('FieldError not raised')
         except FieldError as ex:
             self.assertEqual(str(ex), "Cannot resolve keyword 'pub_date_year' "
-                             "into field. Choices are: author, author_id, headline, "
```

Comment:
I bet it's because author_id will now be a hidden field, so the question is: should hidden fields appear in this list?